frontend/packages/tools/package.json

```diff
@@ -11,7 +11,6 @@
     "lint:fix": "yarn run lint --fix",
     "publishBeta": "../../../tools/publish-frontend.sh tools",
     "registerWorkflows": "node workflow-registrar.js",
-    "test": "jest --passWithNoTests",
```

Comment:
Removed this from tools as with jest 27 its failing with a validation error even with `--passWithNoTests` since `src` does not exist.

```
@clutch-sh/tools: $ jest --passWithNoTests
@clutch-sh/tools: ● Validation Error:
@clutch-sh/tools:   Directory ...frontend/packages/tools/src in the roots[0] option was not found.
@clutch-sh/tools:   Configuration Documentation:
@clutch-sh/tools:   https://jestjs.io/docs/configuration
@clutch-sh/tools: error Command failed with exit code 1.
@clutch-sh/tools: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```